src/reference/asciidoc/graph.adoc

```diff
@@ -12,64 +12,83 @@ A Spring Integration application with only the default components would expose a
 [source,json]
 ----
 {
-  "contentDescriptor": {
-    "providerVersion": "4.3.0.RELEASE",
-    "providerFormatVersion": 1.0,
-    "provider": "spring-integration",
-    "name": "myApplication"
+  "contentDescriptor" : {
+    "providerVersion" : "5.2.0.RELEASE",
```

Comment:
Let's make this based on the `{project-version}` placeholder!
Let's hope `subs="normal"` in the `[source,json]` definition helps!